airflow/www/templates/airflow/dags.html

```diff
@@ -164,7 +164,7 @@ <h2>{{ page_title }}</h2>
                 <td>
                   <a href="{{ url_for('Airflow.'+ dag.get_default_view(), dag_id=dag.dag_id) }}"
                      title="{{ dag.description[0:80] + '…' if dag.description and dag.description|length > 80 else dag.description|default('', true) }}">
-                    <strong>{{ dag.dag_id }}</strong>
+                    <strong>{{ dag.display_name }}</strong>
```

Comment:
```suggestion
                    <strong title="{{ dag.dag_id }}">{{ dag.display_name }}</strong>
```

Let's still expose the `dag_id` on hover.

Reply:
> Let's still expose the `dag_id` on hover.

Cool! Will fix it in the next commit. Thx for the suggestions : )